go.mod

```diff
@@ -236,6 +236,7 @@ require (
 )
 
 replace (
+	github.com/container-orchestrated-devices/container-device-interface => github.com/klihub/container-device-interface v0.0.0-20220224122516-00509aa11be4
```

Comment:
what is the plan for replacing this override? I think it would be best not to merge a fork if possible

Reply:
We have the same change merged already in the upstream repo.